lib/evss/disability_compensation_form/service.rb

```diff
@@ -59,12 +59,40 @@ def validate_form526(form_content)
           headers = { 'Content-Type' => 'application/json' }
           options = { timeout: Settings.evss.disability_compensation_form.submit_timeout || 355 }
           raw_response = perform(:post, 'validate', form_content, headers, options)
-          FormSubmitResponse.new(raw_response.status, raw_response)
+          if raw_response.status == 200
+            valid_response
+          else
+            # when evss returns invalid, the middleware will catch the error and it will be rescued
+            # by the caller. This is the fallback response for an unknown error response
+            error_response
+          end
         end
       end
 
       private
 
+      def valid_response
```

Comment:
do we typically handle the conversion to json-api formatted responses in the service? Seems like something we handle at the controller level